backend/src/test/java/gov/cdc/usds/simplereport/validators/FileValidatorTest.java

```diff
@@ -16,24 +15,28 @@
 
 class FileValidatorTest {
 
+  List<String> patientBulkUploadRequiredFields = PatientUploadRow.requiredFields;
```

Comment:
Accessing the field objects directly like this isn't quite best practice - it's easy to accidentally reassign the values. Instead, you can make the `requiredFields` fields private and add `@Getter` to each (you can also add `setters` if you need to, but I'm not sure that's necessary).

Reply:
when I was taking my first stab at addressing this comment, I totally misread what you wrote so adding the `@Getter` wasn't working for me (I believe because I was implementing it incorrectly haha 😅) so instead I added a new method `getRequiredFields()` to get the fields instead... let me know if this works... If not I'll go back and change it 😅